daisy_integration_tests/scripts/v2v_adapt.sh

```diff
@@ -0,0 +1 @@
+docker run --privileged -v /dev/disk/by-id:/dev/disk/by-id:ro v2v -vv --log file=/dev/ttyS1,level=debug --log file=/dev/ttyS2,level=trace --report file=/dev/ttyS3 --report file=http://metadata.google.internal/computeMetadata/v1/instance/guest-attributes/v2v/report,headers=Metadata-Flavor=Google --drive file=/dev/disk/by-id/google-workload-disk1 --in-place
```

Comment:
Usually you'd see a shebang line, a license, and the long line split up with slashes (\), and termination of the file with a newline. Did you need to omit those to make the `--byol` work below?

Example of the first two: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_integration_tests/scripts/post_translate_test.sh

Reply:
Yes. I could add a parameter to the script, but let's leave that for later as more tests are added as I feel this won't be the final version of this test suite.

Reply:
sgtm.